### DISCUSSION TITLE
How to access revisions table with related collection by id in desc order

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/api#2468] [Q&A]
Hi team,

I am not able to fetch revisions table with related collection by id in desc order, Any idea how to get? I am able fetch only 100 record by end point '/revisions'.
I am using V9.0.1 version.

**Comment by benhaynes** [comment]
Why are you posting this in the version 8 archive repository? Are you using version 9?

**Reply by sandyksit** [reply]
ohh yes i am using version 9, closing this and opening new one
